src/inversion_scripts/calc_sensi.py

```diff
@@ -106,7 +109,10 @@ def process(h):
                 # Get the data for the current hour
                 pert = pert_data["SpeciesConcVV_CH4"][h, :, :, :]
                 # Compute and store the sensitivities
-                sensitivities = (pert.values - base.values) / perturbation
+                if (perturbationBC is not None) and (e > (nelements-4)):
+                    sensitivities = (pert.values - base.values) / (perturbationBC*1e-9)
```

Comment:
When optimizing BCs we need to convert the perturbation into ppb to match units with the values in the simulation, hence the `*1e-9`